docs/upgrading.md

```diff
@@ -16,6 +16,14 @@ $ rake db:migrate
 
 ---
 
+# From v1.0 to v1.3
```

Comment:
I don't think this is needed as consolidating these migrations shouldn't affect anyone and this could potentially cause confusion.